tests/test_runs/test_run_functions.py

```diff
@@ -635,11 +638,13 @@ def test_run_and_upload_knn_pipeline(self):
         n_missing_vals = self.TEST_SERVER_TASK_MISSING_VALS[1]
         n_test_obs = self.TEST_SERVER_TASK_MISSING_VALS[2]
         self._run_and_upload_classification(pipeline2, task_id, n_missing_vals, n_test_obs, "62501")
+        self.assertEqual(warnings_mock.call_count, 3)
 
     def test_run_and_upload_gridsearch(self):
         gridsearch = GridSearchCV(
             BaggingClassifier(base_estimator=SVC()),
             {"base_estimator__C": [0.01, 0.1, 10], "base_estimator__gamma": [0.01, 0.1, 10]},
+            cv=3,
```

Comment:
Is this to reduce load? Is there a reason why you don't also reduce the dimensions (e.g. to 2x2 grid)?

Reply:
No, this is to make sklearn stop printing a warning that the default value will change in the future.